next-level/lib/checkers/python.py

```
@@ -0,0 +1,106 @@
+        line_count = len(lines)
+        if line_count > 500:
+            result["length_warning"] = f"File is {line_count} lines (>500) — consider splitting"
+        elif line_count > 300:
+            result["length_warning"] = f"File is {line_count} lines (>300) — getting long"
+    except (OSError, UnicodeDecodeError):
+        pass
+
+    # Format with ruff
+    if shutil.which("ruff"):
+        try:
+            proc = subprocess.run(
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)